app/views/dashboard/_comment_moderate.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if comment.status == 4 %>
```

Comment:
I think these changes are not related to the issue upon which you are working upon. Can you please remove them?